src/Serilog.Sinks.Seq/SeqLoggerConfigurationExtensions.cs

```diff
@@ -60,7 +60,8 @@ public static class SeqLoggerConfigurationExtensions
             long? bufferFileSizeLimitBytes = null,
             long? eventBodyLimitBytes = 256 * 1024,
             LoggingLevelSwitch controlLevelSwitch = null,
-            HttpMessageHandler messageHandler = null)
+            HttpMessageHandler messageHandler = null,
+            long? errorFilesSizeLimitBytes = null)
```

Comment:
Needs XML documentation (I realised checking this we've missed it for `messageHandler` - we should switch warnings-as-errors on here to make sure these are caught, sorry).